Web_Development/Javascript/Topics/5.Syntax/6.Functions/README.md

```diff
@@ -0,0 +1,96 @@
+## What is scope?
+Scope in javascript manages availability of variables. Let's take an example
+``` javascript
+var a=100 ;
+console.log(a); // 100
+```
+You can easily console log this variable. Let's consider another 
+example
+``` javascript 
+if (true) {
+  const a = 2;
+}
+console.log(a); //ReferenceError
```

Comment:
Mention the error message exactly as it shows in the console window `ReferenceError: a is undefined`